tests/framework/filters/VerbFilterTest.php

```
@@ -0,0 +1,256 @@
+        $event = $this->mockActionEvent('view');
+
+        $filter = new VerbFilter([
+            'actions' => [
+                'view' => ['GET', 'POST'],
+            ],
+        ]);
+
+        try {
+            $filter->beforeAction($event);
+        } catch (MethodNotAllowedHttpException $e) {
+        }
```

PHPMD (2.15.0)

[error] Avoid using empty try-catch blocks in testDisallowedVerbSetsIsValidFalse. (undefined)

(EmptyCatchBlock)